server/testdriver/junit4/src/main/java/org/infinispan/server/test/junit4/InfinispanXSiteServerRule.java

```diff
@@ -0,0 +1,63 @@
+package org.infinispan.server.test.junit4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.infinispan.server.test.core.TestServer;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/**
+ * Creates a cluster of sites to be used for running multiple tests
+ * for Cross-Site replication functionality, It performs the following tasks
+ * <ul>
+ * <li>It creates a temporary directory using the test name</li>
+ * <li>It creates a common configuration directory to be shared by all servers</li>
```

Comment:
no longer true. the configuration directory is shared only for servers from the same site.